<div class="panel panel-default">
    <div class="panel-body">
        <div class="page-header">
            <h1>CoReBIN <small>Comparison and Evaluation of metagenomic binning</small></h1>
        </div>

        <h4>Info</h4>
        <p>
            CoReBIN is a user-friendly web-based tool to compare the results of different binning methods and to aid manual refinement of the bins. The overall binning result of two different methods can be visually compared in a chord diagram. Individual bins can be inspected and refined using GC%, coverage and tetranucleotide frequencies.               
        </p>
        
        <h4>Input data</h4>
        <p>
            <ol>
            <li>Metagenomic assembly (fasta format)
            <br/>
            Coverage table (see format details below) <i>not required</i>             
            </li>
            <li>One or more binning results (see format details below)</li>
            </ol>
        </p>

        <h4>Getting started</h4>
        <p>First, upload your metagenomic assembly in fasta format. It is advised to also upload a table with coverage information on your assembly. Next you can upload the result of your binning. To do so, you need a file in which the contig ID if followed by the bin number (e.g. contig </t> binID).
        </p>
        
        <h4>Metagenomic binning</h4>
        <p>
        Here are some suggestions of recently published metagenomic binning programs and methods:
            <ul>
            <li><a href="https://github.com/BinPro/CONCOCT">CONCOCT</a>: Alneberg, Johannes, et al. "Binning metagenomic contigs by coverage and composition." <i>Nature methods</i> 11.11 (2014): 1144-1146.</li>
            <li><a href="https://bitbucket.org/berkeleylab/metabat">MetaBAT</a>: Kang, Dongwan D., et al. "MetaBAT, an efficient tool for accurately reconstructing single genomes from complex microbial communities." <i>PeerJ</i> 3 (2015): e1165.</i>
            <li><a href="http://ecogenomics.github.io/GroopM/">GroopM</a>: Imelfort, Michael, et al. "GroopM: an automated tool for the recovery of population genomes from related metagenomes." <i>PeerJ</i> 2 (2014): e603.</li>
            <li><a href="https://sourceforge.net/projects/maxbin/">MaxBin 2.0</a>: Wu, Yu-Wei, Blake A. Simmons, and Steven W. Singer. "MaxBin 2.0: an automated binning algorithm to recover genomes from multiple metagenomic datasets." <i>Bioinformatics</i> (2015): btv638.</li>
            <li><a href="http://claczny.github.io/VizBin/">VizBin</a>: Laczny, Cedric C., et al. "VizBin-an application for reference-independent visualization and human-augmented binning of metagenomic data." <i>Microbiome</i> 3.1 (2015): 1.</li>
            </ul>
        </p>           
    </div>
</div>

<div class="panel panel-info">
    <div class="panel-heading"> 
        <span class="fa fa-newspaper-o fa-lg pull-left"></span> Updates
    </div>
    <div class="panel-body">
        *updates*
    </div>
</div>